<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>0</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QCheckBox" name="Out_0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Out_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Out_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Out_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Out_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Out_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>5</string>
    </property>
    <widget class="QRadioButton" name="r_0_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_1_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_2_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_3_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_4_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_5_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>4</string>
    </property>
    <widget class="QRadioButton" name="r_0_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_1_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_2_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_3_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_4_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_5_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>3</string>
    </property>
    <widget class="QRadioButton" name="r_0_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_1_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_2_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_3_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_4_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_5_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>2</string>
    </property>
    <widget class="QRadioButton" name="r_0_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_1_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_2_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_3_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_4_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_5_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>1</string>
    </property>
    <widget class="QRadioButton" name="r_0_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_1_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_2_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_3_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_4_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_5_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>0</string>
    </property>
    <widget class="QRadioButton" name="r_0_0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_1_0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_2_0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_3_0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_4_0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="r_5_0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbGenerate">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>660</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtPatch">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>190</y>
      <width>251</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>401</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="op0_tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="txt_op_0_FRatio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Freq Ratio</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_Att_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Attack time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_Att_Amp">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Attack max amplitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Decay Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_Decay_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Sustain level</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Sustain Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_sus_lvl">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_sus_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Release Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_re_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="op0_cb_fixamp">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>FixAmp</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="txt_op_0_sus_lvl_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_sus_time_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Decay Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Freq Ratio</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_Decay_time_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_FRatio_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Sustain Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_Att_time_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Attack max amplitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Sustain level</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_Att_Amp_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_op_0_re_time_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="op0_cb_fixamp_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>FixAmp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Release Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Attack time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
